test/test_quantized_nn_mods.py

```diff
@@ -110,6 +116,7 @@ def test_linear_api(self, batch_size, in_features, out_features, use_bias):
 
 
 class ModuleAPITest(QuantizationTestCase):
+    @no_deadline
```

Comment:
Wonder what `@no_deadline` mean here? We can run the unit test as long as we want?

Reply:
hypothesis compares test timings across invocations with the same parameter. For various reasons (CUDA runtime initialization, ASAN startup overhead) the first run could take much longer than subsequent runs, but in these cases this is not a useful reason for our tests to fail. So we just disable that check

Reply:
While in this case everything is fine, we should avoid stamps like this when the reviewer doesn't understand what the change does. Please either check the documentation/code or check with the author before approving. Code reviews are there for a reason.

Reply:
I see. Sorry for hurrying. I saw `@unittest.skipIf(` is missed in my previous PR: https://github.com/pytorch/pytorch/pull/23128, so it might cause the OSS failure.